qiskit/optimization/algorithms/grover_optimizer.py

```diff
@@ -345,11 +346,16 @@ def __init__(self, x: Union[List[float], np.ndarray], fval: float, variables: Li
             operation_counts: The counts of each operation performed per iteration.
             n_input_qubits: The number of qubits used to represent the input.
             n_output_qubits: The number of qubits used to represent the output.
+            intermediate_fval: The intermediate value of the objective function of the solution,
+                that is expected to be identical with ``fval``.
+            threshold: The threshold of Grover algorithm.
         """
-        super().__init__(x, fval, variables, None)
+        raw_results = {'intermediate_fval': fval, 'threshold': threshold}
+        super().__init__(x, fval, variables, raw_results)
         self._operation_counts = operation_counts
         self._n_input_qubits = n_input_qubits
         self._n_output_qubits = n_output_qubits
+        self._intermediate_fval = intermediate_fval
```

Comment:
This is unused - I would understand why it would be stored if this and threshold were made available as intermediate result properties here, but they are not

Reply:
Thank you. It is a mistake.